framework/src/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -331,19 +332,48 @@ public boolean implementsWidening() {
         public AnnotationMirror widenUpperBound(AnnotationMirror a1, AnnotationMirror a2) {
             AnnotationMirror lub = leastUpperBound(a1, a2);
             if (AnnotationUtils.areSameByClass(lub, IntRange.class)) {
-                Range range = getRange(lub);
-                if (range.isWithin(Byte.MIN_VALUE, Byte.MAX_VALUE)) {
+                Range lubRange = getRange(lub);
+                if (Range.IGNORE_OVERFLOW) {
+                    Range range1 = getRangeOrConvertIntVal(a1);
```

Comment:
This whole methods needs more thought -- both when ignoring and not ignoring overflow.  I'm going to do that in another pull request, but I don't think that should block this pull request from being merged.  We can either revert this change (and skip the test associated with it) or just leave it as is.